packages/zoe/src/zoeService/zoe.js

```diff
@@ -156,94 +152,82 @@ const makeZoeKit = (
+    },
+    async getFeeIssuer() {
+      return feeMintKit.feeMint.getFeeIssuer();
+    },
 
-        getBrands(instance) {
-          const { state } = this;
-          return state.dataAccess.getBrands(instance);
-        },
-        getIssuers(instance) {
-          const { state } = this;
-          return state.dataAccess.getIssuers(instance);
-        },
-        getPublicFacet(instance) {
-          const { state } = this;
-          return state.dataAccess.getPublicFacet(instance);
-        },
-        getTerms(instance) {
-          const { state } = this;
-          return state.dataAccess.getTerms(instance);
-        },
-        getInstallationForInstance(instance) {
-          const { state } = this;
-          return state.dataAccess.getInstallation(instance);
-        },
-        getBundleIDFromInstallation(installation) {
-          const { state } = this;
-          return state.dataAccess.getBundleIDFromInstallation(installation);
-        },
-        getInstallation,
+    getBrands(instance) {
+      return dataAccess.getBrands(instance);
+    },
+    getIssuers(instance) {
+      return dataAccess.getIssuers(instance);
+    },
+    getPublicFacet(instance) {
+      return dataAccess.getPublicFacet(instance);
+    },
+    getTerms(instance) {
+      return dataAccess.getTerms(instance);
+    },
+    getInstallationForInstance(instance) {
+      return dataAccess.getInstallation(instance);
+    },
+    getBundleIDFromInstallation(installation) {
+      return dataAccess.getBundleIDFromInstallation(installation);
+    },
+    getInstallation,
 
-        getInstance(invitation) {
-          return getInstance(invitation);
-        },
-        getConfiguration,
-        getInvitationDetails,
-        getProposalShapeForInvitation(invitation) {
-          const { state } = this;
-          return state.dataAccess.getProposalShapeForInvitation(invitation);
-        },
-      },
-      feeMintAccessRetriever: {
-        /** @type {() => FeeMintAccess} */
-        get() {
-          // @ts-expect-error type cast
-          return feeMintKit.feeMintAccess;
-        },
+    getInstance(invitation) {
+      return getInstance(invitation);
+    },
+    getConfiguration,
+    getInvitationDetails,
+    getProposalShapeForInvitation(invitation) {
+      return dataAccess.getProposalShapeForInvitation(invitation);
+    },
+  });
+
+  const feeMintAccessRetriever = vivifyFarInstance(
+    zoeBaggage,
+    'FeeMintAccessRetriever',
+    FeeMintAccessRetrieverI,
+    {
+      /** @type {() => FeeMintAccess} */
+      get() {
+        // @ts-expect-error type cast
+        return feeMintKit.feeMintAccess;
       },
     },
   );
 
-  const zoeServices = provide(zoeBaggage, 'zoe', () =>
-    makeZoeService(zoeServiceDataAccess),
-  );
-  zoeService = zoeServices.zoeService;
-  return { zoeServices, setVatAdminService };
+  return harden({
+    zoeServices: {
+      zoeService,
+      feeMintAccessRetriever,
+    },
+    setVatAdminService,
+  });
```

Comment:
I preserved the nested record structure, to avoid wrestling with the fallout of changing that. But could we flatten it? Would it be better?